From 300e3c312f20715f74b0c8c726d1482336908c94 Mon Sep 17 00:00:00 2001
From: Kosta Zertsekel <konszert@marvell.com>
Date: Sat, 13 Apr 2013 23:18:45 +0300
Subject: [PATCH 0572/1825] ALP: Remove unneeded file entry-macro.S

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 7b9ec9aaf42b1e848fac912da6842a63a2fb89c0

Rationale:
Original use of 'entry-macro.S' was to hold assembly code to calculated
flattened/virtual IRQ number. Now it is done through 'gic_handle_irq'
and mach-avantalp/irq.c C code.

Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Change-Id: I6246543d9adf411be24057e44c264f7778c3779b
Reviewed-on: http://vgitil04.il.marvell.com:8080/1571
Reviewed-by: Eran Ben-Avi <benavi@marvell.com>
Tested-by: Eran Ben-Avi <benavi@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-avantalp/include/mach/entry-macro.S |   18 ------------------
 1 files changed, 0 insertions(+), 18 deletions(-)
 delete mode 100755 arch/arm/mach-avantalp/include/mach/entry-macro.S

diff --git a/arch/arm/mach-avantalp/include/mach/entry-macro.S b/arch/arm/mach-avantalp/include/mach/entry-macro.S
deleted file mode 100755
index 9221579..0000000
--- a/arch/arm/mach-avantalp/include/mach/entry-macro.S
+++ /dev/null
@@ -1,18 +0,0 @@
-/*
- * include/asm-arm/arch-mv78xx0/entry-macro.S
- *
- * Low-level IRQ helper macros for Marvell MV78xx0 platforms
- *
- * This file is licensed under the terms of the GNU General Public
- * License version 2.  This program is licensed "as is" without any
- * warranty of any kind, whether express or implied.
- */
-
-#include <mach/hardware.h>
-#include <asm/hardware/entry-macro-gic.S>
-
-	.macro	disable_fiq
-	.endm
-
-	.macro	arch_ret_to_user, tmp1, tmp2
-	.endm
-- 
1.7.5.4

